Explorer/Assets/DCL/Infrastructure/MVC/WindowsStackManager/WindowStackManager.cs

```diff
@@ -68,6 +72,11 @@ public void PopOverlay(IController controller) =>
         public PopupPopInfo PopPopup(IController controller)
         {
             popupStack.Remove(controller);
+
+            foreach (var persistant in persistentStack)
```

Comment:
there could be multiple popups one in front of the other, you should call the focus only if all the popups have been closed